copier.yml

```diff
@@ -126,6 +126,18 @@ generate_requirements_txt:
     Generate requirements.txt from addons manifests and optional overrides in setup.py
     files.
 
+use_ruff:
+  default: no
+  type: bool
+  help: Use ruff and ruff-format instead of flake8, autoflake, pyupgrade, isort, black.
+  when: "{{ odoo_version >= 14.0 }}"
```

Comment:
Upgrade the minimal Copier version to make sure `when` and `default` work reliably.

Reply:
@yajo I set min_copier_version to 9. Is that ok or do you know a better value?

Reply:
Hm that does not work because the tests here seem to be incompatible with copier 9. They should be rewritten to use the CLI I think. That will be for another day. Help welcome.

Reply:
https://github.com/OCA/oca-addons-repo-template/pull/221